_amenities/tennis-information.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Tennis Information

```

Comment:
Should we switch this up to look more like other Klahanie pages - same heading and smaller photo?